test/verify/check-storage-used

```diff
@@ -63,6 +65,7 @@ ExecStart=/usr/bin/sleep infinity
 
         self.content_dropdown_action(1, "Format")
         self.dialog_wait_open()
+        sit()
```

Comment:
This should probably not be in the commit :)

Reply:
Ouch!

Reply:
Fixed.